dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
From http://facebook.github.io/react/downloads.html

"To use React in production mode, set the environment variable NODE_ENV to production (using envify or webpack's DefinePlugin)"

I wouldn't be shocked to find out we're using dev mode in production, though I'm not sure how to validate that